cmd/rook/main.go

```diff
@@ -232,6 +240,14 @@ func setLogLevel() {
 	capnslog.SetGlobalLogLevel(cfg.logLevel)
 }
 
+func logStartupInfo(cmdFlags *pflag.FlagSet) {
+	// log the version number, arguments, and all final flag values (environment variable overrides
+	// have already been taken into account)
+	flagValues := flags.GetFlagsAndValues(cmdFlags)
+	logger.Infof("starting rook %s with arguments '%s'", version.Version, strings.Join(os.Args, " "))
```

Comment:
That log statement is no longer needed @travisn, i've removed it.

I'm OK with keeping the functions in main.go for now @kokhang to keep this PR scoped and focused.